src/scripts/eos-analysis

```diff
@@ -526,6 +526,11 @@ def cmd_sample_pmc(args):
     return eos.sample_pmc(**args_to_dict(args))
 
 
+# Nested sampling
+def cmd_sample_nested(args):
```

Comment:
You have added the function, but there is no subparser for the command. Please have a look how ``parser_sample_pmc`` is handled.